packages/larva-css/src/03-utilities/u-font-size.common.inline.scss

```diff
@@ -41,11 +41,13 @@
 		($font-size-14, desktop,)
 		($font-size-18, desktop,)
 		($font-size-20, desktop)
+		($font-size-22, desktop)
```

Comment:
@laras126 I am not following. You mean directly add css for the font-size instead of adding the font sizes in the 'u-font-size' generator? For instance, instead of adding font-size-22 it in the generator I use the following code:

.lrv-u-font-size-22\@desktop {
	@include pmc-breakpoint( desktop ) {
		font-size: pmc-rem(22);
	}
}

Reply:
No, I mean not adding font-size utilities at all, so that all of the typography is documented in the style guide and built through the tokens. Ideally, there is an `lrv-a-font-*` selector that works with the text and it doesn't need utilities, and if there is an update, the design tokens can be updated for that typography style. 

For example, if `lrv-a-font-primary-s` is the selector for this text style, we should update the token `PRIMARY_S_FONT_SIZE_DESKTOP`to be 22px vs. adding a utility to the library and to the element. 

Does that explanation help?

Reply:
@laras126 Thanks for the clarification. You are right the lrv-a-font-* selector works fine. The font sizes were not needed, I have removed them.